runtime/src/version.rs

```diff
@@ -15,7 +15,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
 	spec_version: 214,
```

Comment:
The spec version also needs to be bumped because of the change from

```rust
type DbWeight = RocksDbWeight;
// to
type DbWeight = ParityDbWeight;
```

in the runtime. There are some weight calculations based on the `DbWeight`, so the result of those calculations will change. (That then means that the behavior of the runtime is observably different after this change, which means increasing the spec version is required).